### ISSUE TITLE
Augmentation apparently doesn't work

### ISSUE THREAD

**Issue by vcjob** [mozilla/DeepSpeech#3032]
- **Have I written custom code (as opposed to running examples on an unmodified clone of the repository)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu18.4
- **TensorFlow installed from (our builds, or upstream TensorFlow)**: yours
- **TensorFlow version (use command below)**: 1.15.2
- **Python version**: 3.6.9
- **Bazel version (if compiling from source)**: idk
- **GCC/Compiler version (if compiling from source)**:-
- **CUDA/cuDNN version**:10.1 7.6
- **GPU model and memory**: 2080ti x2
- **Exact command to reproduce**:
`TF_FORCE_GPU_ALLOW_GROWTH=true python3 DeepSpeech.py --drop_source_layers 1 --alphabet_config_path /ssd/asr_kyrg/alphabet.txt --save_checkpoint_dir /ssd/asr_kyrg/checkpoints/ --load_checkpoint_dir /ssd/asr_kyrg/checkpoints/best_dev-774515 --train_files /ssd/asr_kyrg/train.csv --dev_files /ssd/asr_kyrg/dev.csv --test_files /ssd/asr_kyrg/test.csv --train_cudnn --audio_sample_rate 8000 --train_batch_size 64 --data_aug_features_additive 0.2 --data_aug_features_multiplicative 0.2 --augmentation_speed_up_std 0.2 --augmentation_spec_dropout_keeprate 0.8 --augmentation_freq_and_time_masking --augmentation_freq_and_time_masking_freq_mask_range 5 --augmentation_freq_and_time_masking_number_freq_masks 3 --augmentation_freq_and_time_masking_time_mask_range 2 --augmentation_freq_and_time_masking_number_time_masks 3`

I resume the training from my checkpoint and use batch_size - 64 (I have 2x 2080ti), so the total bs must be 128. I have 11k files in train set, and though I think I made the DS do the audmentation, the number of steps is the same as without. Is it fine, or I did smth wrong with the DeepSpeech.py keys? 
That's what I see:
`I Could not find best validating checkpoint.
I Could not find most recent checkpoint.
I Initializing all variables.
I STARTING Optimization
Epoch 0 |   Training | Elapsed Time: 0:00:51 | Steps: 93 | Loss: 231.074074
Epoch 0 | Validation | Elapsed Time: 0:03:29 | Steps: 2000 | Loss: 231.215172 | Dataset: /ssd/asr_kyrg/dev.csv
`
I wish I do the augmentation since I have pretty low data amount even for the transfer learning.
Thank you!

**Comment by reuben** [comment]
Augmentation changes the data in-place as it is being fed, but because it does random transformations every time it loads a file, it's like having more varied files. The number of steps does not change.
